### PR TITLE
make docker image smaller

### PR DESCRIPTION
**Pull request by circa10a**
optimize docker image, reference openebs/openebs#553

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 FROM ubuntu:16.04
 RUN apt-get update && apt-get install -y kmod curl nfs-common fuse \
-        libibverbs1 librdmacm1 libconfig-general-perl libaio1
+        libibverbs1 librdmacm1 libconfig-general-perl libaio1 && rm -rf /var/lib/apt/lists/*
```
Comment (kmova) [inline package/Dockerfile]: Thanks @circa10a -- this is good. 
Can we make a small change to push this to the next line with "\"

**Comment by circa10a** [comment]
👍  Wooo Hacktoberfest!

**Comment by vharsh** [comment]
@circa10a Are you on twitter?

**Comment by circa10a** [comment]
@vharsh I am not, i set one up years ago, but never got into it. I'm just on linkedin. I keep it classy lol

**Pull request merged**